### PR TITLE
Updating nuget packages

### PR DESCRIPTION
**Pull request by Sanan07**
Adding new Pdb2Pdb package and change to `licence` tag because of warning during the build.

### CODE REVIEW

File: src/package/nuspec/TestPlatform.Build.nuspec [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>Build tasks and targets for running tests with Test Platform</description>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <license>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</license>
```
Comment (Haplois) [inline src/package/nuspec/TestPlatform.Build.nuspec:11]: Can you make sure the built packages will list the license correctly after this change? In its documentation, I couldn't find a way to provide a URL, maybe I missed it.

**Review by Haplois — CHANGES_REQUESTED** [review]
Please fix the `.nuspec` files.

File: src/package/nuspec/TestPlatform.Build.nuspec (hunk 1 above)
Comment (Sanan07) [reply src/package/nuspec/TestPlatform.Build.nuspec:11]: I am looking into this.

File: scripts/PortableToFullPdb.ps1 [hunk 2]
```diff
@@ -16,11 +16,11 @@ $TP_ROOT_DIR = (Get-Item (Split-Path $MyInvocation.MyCommand.Path)).Parent.FullN
...
-$PdbConverterToolVersion = "1.1.0-beta1-62316-01"
+$PdbConverterToolVersion = "1.1.0-beta2-21075-01"
 
 function Locate-PdbConverterTool
 {
-    $pdbConverter = Join-Path -path $TP_PACKAGES_DIR -ChildPath "Pdb2Pdb\$PdbConverterToolVersion\tools\Pdb2Pdb.exe"
+    $pdbConverter = Join-Path -path $TP_PACKAGES_DIR -ChildPath "Microsoft.DiaSymReader.Pdb2Pdb\$PdbConverterToolVersion\tools\Pdb2Pdb.exe"
 
        if (!(Test-Path -path $pdbConverter)) {
        throw "Unable to locate Pdb2Pdb converter exe in path '$pdbConverter'."
```
Comment (Haplois) [inline scripts/PortableToFullPdb.ps1:26]: nit: formating

```suggestion
    $pdbConverter = Join-Path -path $TP_PACKAGES_DIR -ChildPath "Microsoft.DiaSymReader.Pdb2Pdb\$PdbConverterToolVersion\tools\Pdb2Pdb.exe"

    if (!(Test-Path -path $pdbConverter)) {
       throw "Unable to locate Pdb2Pdb converter exe in path '$pdbConverter'."
```

File: src/package/external/external.csproj [hunk 3]
```diff
@@ -45,8 +45,8 @@
       <Version>2.0.2</Version>
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-	<PackageReference Include="Pdb2Pdb">
-      <Version>1.1.0-beta1-62316-01</Version>
+	<PackageReference Include="Microsoft.DiaSymReader.Pdb2Pdb">
```
Comment (Haplois) [inline src/package/external/external.csproj:48]: nit: formatting

```suggestion
    <PackageReference Include="Microsoft.DiaSymReader.Pdb2Pdb">
```

**Review by Haplois — APPROVED** [review]
LGTM with some nitpicking around formatting.

**Pull request merged**